src/main/java/io/appium/java_client/pagefactory/iOSFindBy.java

```diff
@@ -27,7 +27,7 @@
  * element or a list of elements. Used in conjunction with
```

Comment:
@bootstraponline @imurchie 
By.name is deprecated? iOS automation

This change should be reverted anyway